lib/css/components/_stacks-badges.less

```
@@ -105,3 +105,38 @@
 .s-badge__important {
     .badge-styles(transparent, var(--red-600), var(--white));
 }
+
+//  $$  Award Count
+//  ---------------------------------------------------------------------------
+.award-bling {
```

Let's still namespace these.

```suggestion
.s-award-bling {
```